PWGHF/TableProducer/candidateSelectorXicToPKPi.cxx

```diff
@@ -78,7 +78,7 @@ struct HfCandidateSelectorXicToPKPi {
   TrackSelectorPr selectorProton;
   HfHelper hfHelper;
 
-  using TracksSel = soa::Join<aod::Tracks, aod::TracksPidPi, aod::TracksPidKa, aod::TracksPidPr>;
+  using TracksSel = soa::Join<aod::TracksWExtra, aod::TracksPidPi, aod::PidTpcTofFullPi, aod::TracksPidKa, aod::PidTpcTofFullKa, aod::TracksPidPr, aod::PidTpcTofFullPr, aod::pidBayesPi, aod::pidBayesKa, aod::pidBayesPr, aod::pidBayes>;
```

Comment:
Are you using the bayesian PID in the selector? If not I would not subscribe to `aod::pidBayes` and the related tables since this introduce an extra and unneeded dependency in the workflow. Similar consideration for `aod::TracksWExtra`, i.e., `aod::Tracks` should be enough.